recipes/ipumspy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ipumspy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e64249ba3e22a84a99ded0d630dc5977c7b5d464a4074a8407c39c3f2ccc4011
+
+build:
+  number: 0
+  entry_points:
+    - ipums = ipumspy.cli:cli
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - click >=7.0.0,<8.0.0
+    - pandas >=1.0.4,<2.0.0
+    - pip
+    - pyarrow >=3.0.0,<4.0.0
+    - python >=3.6.2
+    - pyyaml >=5.4.1,<6.0.0
+    - requests >=2.25.1,<3.0.0
+    - importlib-metadata >=4.0.1
```

Comment:
You only need all the run dependencies at build time if the package imports itself when building. Is that the case here? If not you can remove all the run dependencies from `host`.

Reply:
@ocefpaf my apologies for the very delayed response. I am not familiar enough with the build process to know whether the package imports itself when building. Certain modules import from other modules within the library at runtime - does this mean the package imports itself when building?

Reply:
Only if it import itself in setup.py. The best way to figure it out is to remove those from here. Let's give it a try.

Reply:
@ocefpaf I removed the run dependencies from the `host` element and was able to build locally and install. Latest commit has that change. Thanks!